331FinalProject.Rmd

```diff
@@ -51,7 +51,7 @@ Take a peak at the first 5 entries
 # setwd("~/Desktop/stat341/R331project/data")
 # setwd("~/School/4A/STAT 331/R331project/data")
 # setwd("~/Desktop/R331project/data")
-# setwd("C:/Users/huawei/Desktop/R331project/data")
+setwd("C:/Users/huawei/Desktop/R331project/data")
 pollutants <- read.csv("pollutants.csv", header = TRUE)
 head(pollutants)
 
```

Comment:
I'd suggest comment out(add '#') to lines 54, 55, 56 here since the master will take care of them~